### PR TITLE
tiny fixes

### PR DESCRIPTION
**Pull request by pieleric**

### CODE REVIEW

File: src/odemis/acq/stream/_base.py [hunk 1]
```diff
@@ -348,11 +348,11 @@ def _va_sync_setter(self, origva, v):
         return: the real new value (as accepted by the original VA)
         """
         if self.is_active.value:  # only synchronised when the stream is active
-            logging.debug("updating VA (%s) to %s", origva, v)
+            logging.debug(u"updating VA (%s) to %s", origva, v)
```
Comment (Sabrina85) [inline src/odemis/acq/stream/_base.py]: might be nice to have all logging debugs consistently starting with a capital letter.

File: src/odemis/acq/stream/_base.py [hunk 2]
```diff
@@ -348,11 +348,11 @@ def _va_sync_setter(self, origva, v):
...
             origva.value = v
             return origva.value
         else:
-            logging.debug("not updating VA (%s) to %s", origva, v)
+            logging.debug(u"not updating VA (%s) to %s", origva, v)
```
Comment (Sabrina85) [inline src/odemis/acq/stream/_base.py]: same here..

File: src/odemis/acq/stream/_base.py [hunk 3]
```diff
@@ -364,7 +364,7 @@ def _va_sync_from_hw(self, lva, v):
         # Don't use the setter, directly put the value as-is. That avoids the
         # setter to again set the Hw VA, and ensure we always accept the Hw
         # value
-        logging.debug("updating local VA (%s) to %s", lva, v)
+        logging.debug(u"updating local VA (%s) to %s", lva, v)
```
Comment (Sabrina85) [inline src/odemis/acq/stream/_base.py]: and here

File: src/odemis/acq/stream/_base.py [hunk 4]
```diff
@@ -269,18 +269,18 @@ def __str__(self):
 
     def _getVAs(self, comp, va_names):
         if not isinstance(va_names, set):
-            raise ValueError("vas should be a set but got %s" % (va_names,))
+            raise ValueError(u"vas should be a set but got %s" % (va_names,))
```
Comment (Sabrina85) [inline src/odemis/acq/stream/_base.py]: Capital letters

File: src/odemis/driver/simulated.py [hunk 5]
```diff
@@ -73,6 +73,7 @@ class Stage(model.Actuator):
     def __init__(self, name, role, axes, ranges=None, **kwargs):
         """
         axes (set of string): names of the axes
+        ranges (dict string -> float,float): min/max of the axis
```
Comment (Sabrina85) [inline src/odemis/driver/simulated.py]: also add other params

Comment (pieleric) [reply src/odemis/driver/simulated.py]: `name` and `role` are just the standard arguments. `kwargs` is just there to pass "hidden" arguments to the base Component. So there are no other arguments to describe.

File: src/odemis/acq/stream/_base.py (hunk 4 above)
Comment (pieleric) [reply src/odemis/acq/stream/_base.py]: As this commit is about changing the messages to unicode, it's orthogonal to the content of the messages. It's nicer to see each commit doing a single thing. However, I've added another commit, which does fix the capital for each message in acq.stream .

**Pull request merged**